### ISSUE TITLE
Build fails after compiling rubrail

### ISSUE THREAD

**Issue by rennerom** [mrmekon/connectr#39]
Hi @mrmekon, I cloned the connectr repository and required dependancies, but the build keeps failing somewhere in the rubrail compilation. Specifically at this point

```
   Compiling rubrail v0.9.0
warning: use of deprecated item 'try': use the `?` operator instead
  --> deps/rustfm-scrobble/src/models.rs:85:45
   |
85 |             let deser_result: json::Value = try!(serde::Deserialize::deserialize(de));
   |                                             ^^^
   |
   = note: `#[warn(deprecated)]` on by default

warning: use of deprecated item 'std::error::Error::description': use the Display impl or to_string()
   --> deps/rustfm-scrobble/src/http.rs:163:40
    |
163 |         Err(x) => { Err(x.utf8_error().description().to_string()) }
    |                                        ^^^^^^^^^^^

warning: 2 warnings emitted
```

**Comment by mrmekon** [comment]
This is now fixed in version 1.1.2.  Thanks!

**Comment by rennerom** [comment]
thank you!! It's running great!
